src/assets/scripts/client/airport/MapLayerModel.js

```diff
@@ -0,0 +1,118 @@
+import _forEach from 'lodash/forEach';
+import _isEqual from 'lodash/isEqual';
+import _isNumber from 'lodash/isNumber';
+import _map from 'lodash/map';
+import BaseModel from '../base/BaseModel';
+import DynamicPositionModel from '../base/DynamicPositionModel';
+import StaticPositionModel from '../base/StaticPositionModel';
+
+/**
+ * A collection of lines that represent a map layer.
```

Comment:
**Soooooo** glad to see this split out!

what, exactly, _is_ a `map layer`?   could you add a little bit more here about: 
* what this is for
* why you would use it
* what consumes this

Reply:
I'll look at other models to figure out the way that info should be formatted